src/components/signin/components/Signin.js

```diff
@@ -52,6 +53,7 @@ class Signin extends React.Component {
 
         if(signinStatus){
             message.success(signinMessage, 5);
+            setCookie();
```

Comment:
This function will not be picked up by redux. You need to grab the action from the props of the component because that action has been bound to dispatch (dispatch is how components alert redux that an action has fired)